models/pathology_nuclei_classification/configs/inference_trt.json

```diff
@@ -1,12 +0,0 @@
-{
```

Comment:
I would suggest to keep this `inference_trt.json` file and put all your `trt_wrap` logic here.
Then users still run the previous behavior:
`python -m monai.bundle run --config_file "['configs/inference.json', 'configs/inference_trt.json']"`
What do you think?

Thanks.

Reply:
Well, from the developer perspective, this is kind of clean unintrusive way to specify TRT support, and most of inference_trt.json are going to be identical:
... imports and trt_args here ... 
"network": "$trt_wrap(@network_def.to(@device), @checkpoint_path, @trt_args)"

From the user perspective,  especially if TRT-accelerated inference is a default, it may be simpler to have one config and --trt switch.  

I am fine with either approach.

Reply:
Thanks @borisfom .
MONAI is a community OSS project, not only supporting NVIDIA GPUs, so we don't enable TensorRT by default, `inference_trt.json` can help isolate this advanced acceleration feature.
What do you think?

Thanks.

Reply:
If it's not on by default, then from user perspective separate inference_trt.json is close to --trt flag, and better in terms of source structure. Let's go with that, sure.